ui/js/controllers/logviewer.js

```diff
@@ -376,6 +401,21 @@ logViewerApp.controller('LogviewerCtrl', [
             });
         }
 
+        function displayErrorLines(errors) {
+            var css = errors
+              .reduce((line, err) => `${line}a[id="${err.line_number + 1}"]+span, `, '')
+              .slice(0, -2)
```

Comment:
@eliperelman it slices the last space character with the last comma. But now that you pointed this out, i can simply not add a space between each selector and simply slice the last comma before doing a `concat` with the style object. Thoughts?